staging/src/k8s.io/kubectl/pkg/explain/v2/templates/openapiv3.tmpl

```diff
@@ -0,0 +1,13 @@
+{{ if not (empty .FieldPath) }}
```

Comment:
I don't know what to think of this. This is exposing a new API and I don't know if I agree with that plan.

Reply:
removed this template